xml/System.Reflection/Assembly.xml

```diff
@@ -3076,7 +3076,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Is Assembly held in a collectible AssemblyLoadContext?</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if this assembly is held in a collectible <see cref="T:System.Runtime.Loader.AssemblyLoadContext" />; otherwise, <see langword="false" />.</value>
```